bokeh/_json_encoder.py

```diff
@@ -79,4 +79,4 @@ def default(self, obj):
             return self.transform_python_types(obj)
 
 def serialize_json(obj, encoder=BokehJSONEncoder, **kwargs):
-    return json.dumps(obj, cls=encoder, allow_nan=False, **kwargs)
+    return json.dumps(obj, cls=encoder, allow_nan=True, **kwargs)
```

Comment:
Yes it is, see https://github.com/bokeh/bokeh/pull/2971/files#r41716985 This will work for static docs, but not for server ones that require using `JSON.parse` I think we will have to make custom JS reviver to handle `NaN` and `Infinity`.